src/game/Tactical/Points.cc

```diff
@@ -1678,7 +1678,7 @@ INT16 MinAPsToThrow(SOLDIERTYPE const& s, GridNo gridno, bool const add_turning_
 	const ItemModel *item = GCM->getItem(in_hand);
 	// Gennady: This is a very strange piece of code.
 	//          Be very careful with it.
-	if (!item->getItemClass() & IC_GRENADE)
+	if (!(item->getItemClass() & IC_GRENADE))
```

Comment:
Ahh, the comment made me remember this is not a good fix. The discussion is in #287 and it's more likely this whole block should be removed or redone.

Reply:
Changed to conserve previous behavior and mention the issue.